tutorial/getting-started-aaos.md

```diff
@@ -0,0 +1,64 @@
+This tutorial guides you through basic operations that you can do with an application based on an Android Automotive OS (AAOS) image. 
+
+In this tutorial, we will be specifically focusing on the following tasks:
+- Creating an application using an AAOS image
+- Stream the application you created with a specific screen resolution
+- Set the temperature of the automotive using the Vehicle Hardware Abstraction Layer (VHAL) controls
+
+## Prerequisites
+We will use the appliance dashboard to perform the tasks for this tutorial. So if you haven’t installed the Anbox Cloud Appliance yet, you must do so before you can continue with this tutorial. Follow the installation instructions available at [How to install the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/22681) to install and initialise the appliance.
+
+## Create an AAOS-based application
+
+On the web dashboard *Applications* screen, click *Add Application*.
+
+Enter values for the mandatory fields, keeping the following points in mind:
+- Select *Automotive* as the application type.
+- Select an AAOS image. The image name should indicate whether it is an AOSP or an AAOS image. For example, `jammy:aaos13:amd64` is an AAOS image.
+
+Click *Add application* and wait till the *Applications* list screen shows your application with a ready status.
+
+## Stream the application with a specific screen resolution
+
+On the *Sessions* screen, click *Create session*.
+
+Select the application that you created earlier.
+
+Any of the *Screen resolution* options listed for the application should work but to set a custom resolution, select *Custom* from the drop-down and set the following values:
+```
+Width = 2304
+Height = 3072
```

Comment:
This seem quite large, especially for software rendering/encoding this will cause high CPU load if no GPU is involved. What was the reason to choose this resolution?

Reply:
It's set to that amount because we cannot set the DPI. If we stick with lower resolutions, it becomes impossible to use the UI.

Reply:
For instance: 
![image](https://github.com/canonical/anbox-cloud-docs/assets/98706738/dade1deb-c0f9-43e3-a05a-267e848651bd)

Or: 
![image](https://github.com/canonical/anbox-cloud-docs/assets/98706738/8259cb17-8317-49b4-8d74-2ddbf2d7cfdf)



We could ask the UI team to change the default resolutions available, but the two that we have correspond to actual vehicle screens, so we thought they were good examples, hence why we just took one resolution and doubled it.

Reply:
Hm, that is unfortunate. Once https://github.com/canonical/ams/pull/75 lands we should refine this and use a lower resolution. Fine to keep what we have for now.